glean-core/ffi/src/labeled.rs

```diff
@@ -30,10 +32,8 @@ use crate::*;
 /// * `get_name` - Function name to get a new instance of the underlying metric.
 macro_rules! impl_labeled_metric {
     ($metric:ty, $global:ident, $metric_global:ident, $new_name:ident, $destroy_name:ident, $get_name:ident, $test_get_num_recorded_errors:ident) => {
-        lazy_static! {
-            static ref $global: ConcurrentHandleMap<LabeledMetric<$metric>> =
-                ConcurrentHandleMap::new();
-        }
+        static $global: Lazy<ConcurrentHandleMap<LabeledMetric<$metric>>> =
+            Lazy::new(|| ConcurrentHandleMap::new());
```

Comment:
If we use the full type here we won't need the import in all other files.
We can also avoid allowing that lint.

```suggestion
        static $global: once_cell::sync::Lazy<ConcurrentHandleMap<LabeledMetric<$metric>>> =
            once_cell::sync::Lazy::new(ConcurrentHandleMap::new);
```